src/content/resources/language/evolution.md

```diff
@@ -42,6 +42,19 @@ on the Dart language GitHub repo.
 
 ## Changes in each release
 
+### Dart 3.6
+_Released 11 December 2024_
+| [Dart 3.6 announcement](https://medium.com/dartlang/announcing-dart-3-6-778dd7a80983)
+
+Dart 3.6 added the [digit separator][] underscore (`_`) to the language.
```

Comment:
This wording makes it seem like "the [...] underscore" is a language construct, but rather number literals are.

Would something like the following make sense, so it indicates there can be more than one?

```suggestion
Dart 3.6 added support for [digit separator][] underscores (`_`) to the language.
```

Reply:
That's a really good point, thank you!